pyleecan/Methods/Mesh/MeshSolution/get_group.py

```diff
@@ -93,47 +94,16 @@ def get_group(self, group_names):
     # 4) select the corresponding solutions
     sol_list = list()
     for sol in self.solution:
-        label_sol = sol.label
         type_cell_sol = sol.type_cell
-        field_sol = sol.get_field()
-        axis_name, axis_size = sol.get_axes_list()
 
         if type_cell_sol == "point":
-            Iindice = np.where(axis_name == "indice")[0]
-            axis_size[Iindice] = len(node_indice)
-
-            new_field_sol = field_sol[:, node_indice, :]
-            new_sol = SolutionMat(
-                label=label_sol,
-                type_cell=type_cell_sol,
-                field=new_field_sol,
-                indice=node_indice,
-                axis_name=axis_name,
-                axis_size=axis_size,
-            )
-            sol_list.append(new_sol)
+            new_indices = node_indice
```

Comment:
It was probably an error from previous modification, but it is the 1st occurence of node_indice.

Reply:
node_indice is set in the 'for sep in sep_list:' loop starting in line 62 and in the 'if is_interface:' branch starting in line 85.

Reply:
You are right, it was folded in the Github view. Sorry ;-)